tools/build_pytorch_libs.bat

```diff
@@ -175,6 +175,9 @@ goto:eof
   cmake .. %CMAKE_GENERATOR_COMMAND% ^
                   -DCMAKE_BUILD_TYPE=%BUILD_TYPE% ^
                   -DBUILD_CAFFE2=OFF ^
+                  -DBUILD_TORCH="%BUILD_TORCH%" ^
+                  -DNVTOOLEXT_HOME="%NVTOOLEXT_HOME%" ^
+                  -DNO_API=ON ^
```

Comment:
@goldsborough Is this intentional?

Reply:
the API was only ever compiled through cpp_build, not through setup.py, and cpp_build had never worked on windows

Reply:
OK, but morally, there isn't any reason why C++ API shouldn't work on Windows, except maybe because we're not actively supporting at the moment. Deserves a comment, at least.

Reply:
I'll open an issue when I merge